stdpopsim/cli.py

```diff
@@ -499,40 +587,112 @@ def add_simulate_species_parser(parser, species):
     )
 
     def run_simulation(args):
-        if args.demographic_model is None:
-            model = stdpopsim.PiecewiseConstantSize(species.population_size)
-            model.generation_time = species.generation_time
-            for citation in species.citations:
-                reasons = {stdpopsim.CiteReason.POP_SIZE, stdpopsim.CiteReason.GEN_TIME}
-                if len(citation.reasons & reasons) > 0:
-                    model.citations.append(citation)
-            qc_complete = True
+        if args.dfe is None:
+            dfe = None
+            if args.demographic_model is None:
```

Comment:
It looks like this bit of code defining the demographic model and contig is duplicated - can't we put the `if args.dfe is None:` bit after the contig is defined?

Reply:
you are right! :S thanks!

Reply:
done